src/Conversion/KrnlToLLVM/ConvertKrnlToLLVM.cpp

```diff
@@ -515,13 +516,16 @@ bool extractConstantsToFile(ModuleOp &module, std::string filepath,
     return (leftAlign < rightAlign);
   });
 
-  // Pack all constants into a single buffer in order to save to file.
+  // Store each constant into single file.
   // Constants with the highest alignment will be packed first in the file.
   // The file will be mmaped later at runtime and aligned at the page boundary,
   // So every constants must be correctly aligned in the packed constant. Pads
   // are added if necessary.
-  std::vector<char> packedConst;
+  llvm::sys::fs::remove(filepath);
+  std::ofstream outfile(filepath, std::ios::app | std::ios::binary);
+  int64_t totalConstSize = 0;
   for (int64_t i = globalOfInterest.size() - 1; i >= 0; --i) {
+    std::vector<char> packedConst;
```

Comment:
The name `packedConst` is obsoleted in the new context since we donnot pack constants anymore. Maybe change it to `paddedConstant` or it can be totally removed (See the below comment)